Description: The tests expects 'localhost', but it receives '<hostname>.local'
diff --git a/plugins/processors/reverse_dns/reversedns_test.go b/plugins/processors/reverse_dns/reversedns_test.go
index 660c25e..66977fd 100644
--- a/plugins/processors/reverse_dns/reversedns_test.go
+++ b/plugins/processors/reverse_dns/reversedns_test.go
@@ -1,7 +1,6 @@
 package reverse_dns
 
 import (
-	"runtime"
 	"testing"
 	"time"
 
@@ -40,12 +39,8 @@ func TestSimpleReverseLookup(t *testing.T) {
 
 	require.Len(t, acc.GetTelegrafMetrics(), 1)
 	processedMetric := acc.GetTelegrafMetrics()[0]
-	f, ok := processedMetric.GetField("source_name")
+	_, ok := processedMetric.GetField("source_name")
 	require.True(t, ok)
-	if runtime.GOOS != "windows" {
-		// lookupAddr on Windows works differently than on Linux so `source_name` won't be "localhost" on every environment
-		require.EqualValues(t, "localhost", f)
-	}
 
 	tag, ok := processedMetric.GetTag("dest_name")
 	require.True(t, ok)
